File: app/src/main/java/com/witype/Dragger/mvp/contract/IBaseView.java
package com.witype.Dragger.mvp.contract;

import android.app.Activity;

public interface IBaseView {

    void showToast(String message);

    void showToast(int resId);

    void showLoading(String message);

    void showLoading();

    void dismissLoading();

    Activity getActivity();
}

File: kotlindemo/src/main/java/com/witype/kotlindemo/mvp/contract/IBaseView.kt
package com.witype.kotlindemo.mvp.contract

import android.app.Activity

interface IBaseView {
    fun showToast(message: String)

    fun showToast(resId: Int)

    fun showLoading(message: String)

    fun showLoading()

    fun dismissLoading()

    val activity: Activity?
}